python/variable.cpp

```diff
@@ -187,6 +187,12 @@ void init_variable(py::module &m) {
            py::arg("variances") = std::nullopt,
            py::arg("unit") = units::Unit(units::dimensionless),
            py::arg("dtype") = py::none())
+      .def("rename_dims",
```

Comment:
I know it's only a small amount of duplicated code, but is it possible to refactor so that `Variable`, `Dataset` and `DataArray` all use the same code?